run/run.py

```diff
@@ -127,65 +133,96 @@ def main(platform_id, platform, args, config):
     print('==================================================')
     print('Running WPT')
 
-    if platform.get('sauce'):
-        if platform['browser_name'] == 'edge':
-            sauce_browser_name = 'MicrosoftEdge'
+    for this_chunk in range(1, args.total_chunks + 1):
+        if platform.get('sauce'):
+            if platform['browser_name'] == 'edge':
+                sauce_browser_name = 'MicrosoftEdge'
+            else:
+                sauce_browser_name = platform['browser_name']
+
+            command = [
+                './wpt', 'run', 'sauce:%s:%s' % (
+                    sauce_browser_name, platform['browser_version']),
+                '--sauce-platform=%s' % platform['os_name'],
+                '--sauce-key=%s' % config['sauce_key'],
+                '--sauce-user=%s' % config['sauce_user'],
+                '--sauce-connect-binary=%s' % config['sauce_connect_path'],
```

Comment:
Since we're shaking things up... what do you think about changing `config['sauce_connect_path']` and its corresponding key in runner.ini to `sauce_connect_binary`? We can do that in another PR if you want.

Reply:
That's a good idea. I've also noticed some inconsistency with the treatment of the Firefox and Chrome binaries. As you've pointed out, though, there are no tests for this patch, so I'm inclined to defer those improvements